src/metrics/elasticsearch_metric.ts

```
@@ -0,0 +1,64 @@
+import { AbstractMetric, Datasource, MetricId, MetricQuery, MetricResults  } from './metric';
+
+import * as _ from 'lodash';
+
+
+export class ElasticsearchMetric extends AbstractMetric {
+  constructor(datasource: Datasource, targets: any[], id?: MetricId) {
+    super(datasource, targets, id);
+  }
+
+  getQuery(from: number, to: number, limit: number, offset: number): MetricQuery {
+    let data = this.datasource.data.split('\n').map(d => d !== '' ? JSON.parse(d) : d);
+
+    data[1].size = limit;
```

please throw an exception if `data.length === 0`